UefiPayloadPkg/UefiPayloadPkg.dsc

```diff
@@ -610,7 +613,11 @@
     !if $(UNIVERSAL_PAYLOAD_FORMAT) == "ELF"

```

Comment:
So ELF payload still assumes that boot loader will pass HOBs and not UPL style FDT? Would be good to document all this in readme somewhere.

Reply:
that's right, because it is mainly for backward compatibility. We will document it.